react/features/participants-pane/components/native/ClearableInput.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { View, TextInput, TouchableOpacity } from 'react-native';
+
+import { Icon, IconCloseSolid } from '../../../base/icons';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * If the input should be focused on display.
+     */
+    autoFocus?: boolean,
+
+    /**
+     * Custom styles for the component.
+     */
+    customStyles?: Object,
+
+    /**
+     * Callback for the onChange event of the field.
+     */
+    onChange: Function,
+
+    /**
+     * Callback to be used when the user hits Enter in the field.
+     */
+    onSubmit?: Function,
+
+    /**
+     * Placeholder text for the field.
+     */
+    placeholder: string,
+
+    /**
+     * The type of the return key.
+     */
+    returnKeyType?: 'done' | 'go' | 'next' | 'search' | 'send' | 'none' | 'previous' | 'default',
+
+    /**
+     * Externally provided value.
+     */
+    value?: string
+};
+
+/**
+ * Implements a pre-styled clearable input field.
+ *
+ * @param {Props} props - The props of the component.
+ * @returns {ReactElement}
+ */
+function ClearableInput({
```

Comment:
We have the same use case in the AddPeopleDialog. Please check if it makes sense to have something they can both share.